src/components/leaflet/LeafletMap.jsx

```diff
@@ -1,7 +1,20 @@
 import { MapContainer, TileLayer, Marker } from 'react-leaflet';
 import style from './leaflet.module.scss';
+import { Icon } from 'leaflet';
+import MarkerIcon from '../../assets/images/icon-location.svg';
+import ShadowIcon from '../../assets/images/marker-shadow.png';
 
 const LeafletMap = (props) => {
+  const customIcon = new Icon({
+    iconUrl: MarkerIcon,
+    iconSize: [42, 52],
+    iconAnchor: [16, 32],
+    popupAnchor: [0, -32],
+    shadowUrl: ShadowIcon,
+    shadowSize: [41, 41],
+    shadowAnchor: [12, 41],
+  });
```

Comment:
Looks like a static object without dependencies on props or state. So it has to be moved outside of component to be created just once. Now it's created every re-render.

Reply:
moved